lnbits/core/views/generic.py

```diff
@@ -162,77 +161,46 @@ async def extensions_install(
 @core_html_routes.get(
     "/wallet",
     response_class=HTMLResponse,
-    description="""
-just **wallet_name**: create a new user, then create a new wallet
-                      for user with wallet_name
-just **user_id**: return the first user wallet or create one if none found
-                  (with default wallet_name)
-**user_id** and **wallet_name**: create a new wallet for user with wallet_name
-**user_id** and **wallet_id**: return that wallet if user is the owner
-nothing: create everything
-""",
+    description="show wallet page",
```

Comment:
I hope people are not using this HTML serving endpoint as an REST API one

Reply:
Maybe a similar POST endpoint should exist

Reply:
we have to annouce it a pbreaking change for `0.11.0`